resources/deu.ini

```diff
@@ -1,208 +1,219 @@
 ; German language pack for MyPlot
 ; Created by SalmonDE and morpheus132
-; Updated time : 12/20/2016
+; Updated time : 07/22/2017
 
 ; Popup when entering a plot
-popup=Du hast den Plot {%0} betreten
-popup.owner=Dieser Plot gehört {%0}
-popup.available=Dieser Plot ist verfügbar! Benutze /p claim um den Plot zu beanspruchen!
-popup.denied=Du darfst diesen Plot nicht betreten!
+popup=Du hast das Grundstück {%0} betreten.
+popup.owner=Dieses Grundstück gehört {%0}.
+popup.available=Dieses Grundstück ist verfügbar! Benutze /p claim, um das Grundstück zu beanspruchen.
 
 ; /p info
 command.name=plot
 command.alias=p
-command.desc=Verwalte dein(e) Plot(s)
-command.usage=Usage: /plot <help|sub-cmd> [optionen]
-command.unknown=Unbekannter Befehl! Benutze /p help um eine Liste von Befehlen zu erhalten!
+command.desc=Verwalte deine Grundstücke.
+command.usage=Syntax: /plot <help|Unterbefehl> [Optionen]
+command.unknown=Unbekannter Befehl. Benutze /p help, um eine Liste von Befehlen zu erhalten.
 
-subcommand.usage=Usage: {%0}
+subcommand.usage=Syntax: {%0}
 
 ; Errors that are displayed when the player does something wrong, or the plugin screws up.
 error=Ups! Irgendetwas ist falsch gelaufen!
-notinplot=Du stehst nicht in einem Plot
-notowner=Dieser Plot gehört dir nicht!
+notinplot=Du befindest dich nicht in einem Grundstück.
+notowner=Dieses Grundstück gehört dir nicht.
 
 confirm=confirm
 
 ; The /p help Subcommand
 help.name=help
-help.desc=Zeigt dir das Menü mit allen Befehlen an
-help.usage=/p help [seite]
-help.header=--- Zeige MyPlot Hilfe Seite {%0} of {%1} ---
+help.desc=Zeigt dir das Hilfemenü an.
+help.usage=/p help [Seite]
+help.header=--- Zeige MyPlot Hilfeseite {%0} von {%1} ---
 
 ; The /p addhelper Subcommand
 addhelper.name=addhelper
 addhelper.alias=addh
-addhelper.desc=Füge einen Helfer zu deinem plot hinzu
-addhelper.usage=/p addhelper <spieler>
-addhelper.alreadyone={%0} ist bereits ein Helfer von diesem Plot!
-addhelper.success={%0} wurde zum Helfer von diesem Plot ernannt!
+addhelper.desc=Füge einen Helfer zu deinem Grundstück hinzu.
+addhelper.usage=/p addhelper <Spieler>
+addhelper.alreadyone={%0} ist bereits ein Helfer von diesem Grundstück!
```

Comment:
Anscheinend können Spieler mit der Berechtigung auch Helfer setzen, wenn das Grundstück nicht ihnen gehört. In diesem Fall würde ich nochmal die Beschreibung verändern -> "Füge einen Helfer zu **einem** Grundstück hinzu."